templates/CRM/Event/Form/Registration/ThankYou.tpl

```diff
@@ -179,7 +179,7 @@
         <a href="{crmURL p='civicrm/event/info' q="reset=1&id=`$event.id`"}"><i class="crm-i fa-chevron-left" role="img" aria-hidden="true"></i> {ts 1=$event.event_title}Back to "%1" event information{/ts}</a>
     </div>
 
-    {if $event.is_public and $event.is_show_calendar_links}
+    {if $event.is_active and $event.is_show_calendar_links}
       <div class="action-link section iCal_links-section">
```

Comment:
Why check `is_active` ? If you are seeing the ThankYou page, it's because it's active?

(you mentioned in your PR overview that you were removing these checks, which you do elsewhere, and I think that's OK)